### ISSUE TITLE
Progression bar / time not working

### ISSUE THREAD

**Issue by frozywozy** [MarcoPixel/SysDash#52]
Hello,

I am using Spotify (exe app) and I changed this parameter in the Media (big) file.

IfMatch=WebNowPlaying

It now correctly displays the artist and the title. This being said, the time of the song and the progression bar is still not working. Do I need to change anything in the following parameters?

[MeterPositionDuration]
[MeterProgress]

Thank you.

**Comment by frozywozy** [comment]
Anyone can help?

**Comment by Flanisch** [comment]
> Anyone can help?

Yeah, I've written a script to fix Spotify support. I've posted the files as well as an installation process under a [different issue](https://github.com/marcopixel/SysDash/issues/41#issuecomment-571276056).

I hope this is able to help.
